src/main/scala/cromwell/util/EnhancedExponentialBackoff.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+class EnhancedExponentialBackoff(enhancedBuilder: EnhancedExponentialBackoff.Builder) extends ExponentialBackOff(enhancedBuilder) {
+  private var firstCall: Boolean = true
```

Comment:
Btw this kind of duplicates `TryUtil.retryBlock`. We probably should pick one and use it  everywhere.